protocols/directfund/directfund_test.go

```diff
@@ -24,7 +24,6 @@ const TEST_CHAIN_ID = 1337
 var alice, bob testactors.Actor = testactors.Alice, testactors.Bob
 
 var testState = state.State{
-	ChainId:           big.NewInt(TEST_CHAIN_ID),
```

Comment:
You could also remove the `TEST_CHAIN_ID` constant on L22 (there may be a few others floating around, too?)

Reply:
This const (and others still floating around, like the ENV variables) is used in methods that bubble out to the client API.

Currently, there's no way for the client to specify a chainID (or adjudicator address) at run-time, but presumably that would need to exist in some form or another, and the entry points for chainID would still need to be there.

In effect: taking these out would be doing some work that would eventually need to be re-done.